add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
Observations: The code directly uses user inputs in SQL queries, which exposes the application to SQL injection attacks. Additionally, the database connection is handled in a procedural manner, which is not ideal for maintainability and scalability.

Recommendations: Implement prepared statements to handle user inputs safely. Use a more modern database abstraction layer like PDO or MySQLi with OOP. Consider refactoring the database logic into a separate class to adhere to the Single Responsibility Principle.